docs/safemode.md

```diff
@@ -0,0 +1,58 @@
+# Safemode
+
+Safemode represents a number of safety nets we have implemented into the chaos controller to help new and experienced users feel more confident deploying new disruptions to their environments.
+
+The Chaos Controller can be scary to use in production environments, but we learn a lot more running chaos experiments in our production environments. Considering this, having safety nets in place makes the entire process of running chaos experiments in high value environments a little safer.
+
+The disruption config has a parameter appropriately named `safeMode` which represents a spec containing several boolean fields.
+One of the fields, `IgnoreAll` overrides the rest if set to true. It will ignore every safety net.
+We recommend that `IgnoreAll` be set on the cluster level by default, so maintainers of the controller can turn it on by default in production and maybe off in staging environments.
+When `IgnoreAll` is turned off, the user will have the ability to turn specific safety nets off if they are confident that specific ones are not needed for their use case.
+Please take a look at the example below to see how to format Safemode in your next disruption.
+
+## Ignoring Safety Nets
+
+Because the list of safety nets to be implemented will grow in the future, there will surely be overlap with safety nets which will make it difficult for a user who is confident a specific safety net is not necessary but unsure if others will be.
+Therefore the controller allows for you to ignore specific safety nets in the Safemode Spec. Checkout out example below to see how to remove certain safety nets.
+Remember that these fields to turn off specific safety nets is only accessible to the user if the `IgnoreAll` field is false. Otherwise it overrides all other safety nets and turns them off.
+Keep in mind that all safety nets are turned on by default when safemode is on (`IgnoreAll` set to False), so all that is necessary to ignore safety nets.
+
+### Safety Nets
+
+| Safety Net  | Type | Description | IgnoreName |
+| ----------- | ----------- | ----------- | ----------- |
+| Namespace-wide Targeting                                  | Generic |  Using generical label selectors (e.x. X) that selects a majority of pods/nodes in a namespace as a target to inject a disruption into                     | ignoreCountToLarge        |
+| Sporadic Targets                                          | Generic |  In a volatile environment where targets are being terminated and created sporadically, disruptions should not be allowed to continue disrupting           | ignoreSporadicTargets     |
```

Comment:
I feel this is too situational and shouldn't be enabled by default

Reply:
I think there are two options here; We could have disabled by default but it is unlikely any one will ever use it, the safety nets are all enabled by default because the idea is that they are unforeseen events. And if they are seen, they can be ignored. We can also just remove it if it brings no real value. I'd rather the later as I dont like the idea of having specific safety nets disabled by default, they should all be enabled if `IgnoreAll` is false, and then users can start disabling once they realize which ones are not needed.